AsyncDisplayKit/ASImageNode.mm

```diff
@@ -181,7 +183,8 @@ - (NSObject *)drawParametersForAsyncLayer:(_ASDisplayLayer *)layer
                                                     opaque:self.opaque
                                              contentsScale:self.contentsScaleForDisplay
                                            backgroundColor:self.backgroundColor
-                                               contentMode:self.contentMode];
+                                               contentMode:self.contentMode
```

Comment:
This issue is related to locking issue and crashes seen in `ASTextNode` (and I guess it will not be different for `ASImageNode` as it uses the same mechanism now): https://github.com/facebook/AsyncDisplayKit/issues/1703